src/SFA.DAS.ApprenticeFeedback.Application/Services/ApprenticeFeedbackService.cs

```diff
@@ -50,19 +50,39 @@ public async Task<IEnumerable<Domain.Models.Feedback.TrainingProvider>> GetTrain
                     SignificantDate = tp.SignificantDate,
                 });
             }
-            catch(Exception ex)
+            catch (Exception ex)
```

Comment:
Did we say we were going to delete try/catch blocks that don't do anything other than rethrow the exception?